src/components/survey/survey-reducer.ts

```diff
@@ -7,7 +7,33 @@ import {cioIdentify} from 'utils/cio-identify'
 const DEFAULT_FIRST_QUESTION = `biggest_path`
 const DEFAULT_FINAL_QUESTION = `thanks`
 
-export type SortingHatState = {
+export type Survey = {
+  [key: string]: SurveyQuestion | string
```

Comment:
the `| string` here is because a `Survey` has a version. Not stoked on this for typing but it allows for arbitrary keys of type `SurveyQuestion` so we can build out datasets for surveys